lib/stack-layers.js

```diff
@@ -86,10 +88,11 @@ module.exports = function(element, id, side, layers, drills, outline, useOutline
     var cfMaskId = idPrefix + 'cf-mask'
     var cfMaskAttr = {id: cfMaskId, fill: '#fff', stroke: '#fff'}
     var cfMaskShape = smLayerId
-      ? [useLayer(element, smLayerId)]
+      ? [element('use', {'xlink:href': '#' + smLayerId, fill: '#fff', stroke: '#fff'})]
```

Comment:
Can you jog my memory with these fill and stroke attributes? Were they really necessary? It seems weird to me that specifying them in the parent (i.e. the mask node) wouldn't be enough. Although maybe it makes a difference that now a `<g>` is the parent.

Could you explore (here and below at line 108) keeping this as a `useLayer` call and placing the fill and stroke attributes on the `<g>` element?